app/react/src/server/__tests__/cra-config.test.js

```diff
@@ -1,5 +1,6 @@
 import fs from 'fs';
-import { getReactScriptsPath } from './cra-config';
+import { getReactScriptsPath, getTypeScriptRules } from '../cra-config';
```

Comment:
let's not move the tests into the `__tests__` dir, we have this in a few places, but it's mostly the inconsistency, rather the desired way (IMO).

Reply:
This is now resolved, thanks for the feedback.